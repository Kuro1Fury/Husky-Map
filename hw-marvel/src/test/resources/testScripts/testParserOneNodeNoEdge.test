# Test the parser on a graph that contains no edge

LoadGraph chars oneNodeNoEdge.csv
ListChildren chars Mario