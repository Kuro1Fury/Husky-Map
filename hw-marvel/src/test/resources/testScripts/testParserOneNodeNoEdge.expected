# Test the parser on a graph that contains no edge

loaded graph chars
the children of Mario in chars are: